### ISSUE TITLE
Error during install on node 6 due to outdated optipng-bin and jpegtran-bin dependencies

### ISSUE THREAD

**Issue by charlesdg** [niftylettuce/express-cdn#102]
express-cdn is not compatible with node 6, giving the following error with `optipng-bin` and `jpegtran-bin`. 

```
npm ERR! Darwin 15.5.0
npm ERR! argv "/Users/charlesdegroote/.nvm/versions/node/v6.2.0/bin/node" "/Users/charlesdegroote/.nvm/versions/node/v6.2.0/bin/npm" "install"
npm ERR! node v6.2.0
npm ERR! npm  v3.8.9
npm ERR! code ELIFECYCLE

npm ERR! optipng-bin@0.3.11 postinstall: `node index.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the optipng-bin@0.3.11 postinstall script 'node index.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the optipng-bin package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node index.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs optipng-bin
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls optipng-bin
npm ERR! There is likely additional logging output above.
```

I tried to update the two dependencies but express-cdn is not compatible with them.

**Comment by gagalago** [comment]
duplicate as #101

**Comment by gagalago** [comment]
fixed in 0.2.9
